gpflow/models/gplvm.py

```diff
@@ -47,7 +50,7 @@ def __init__(self, Y, latent_dim, X_mean=None, kern=None, mean_function=None, **
 
 
 class BayesianGPLVM(GPModel):
-    def __init__(self, X_mean, X_var, Y, kern, M, Z=None, X_prior_mean=None, X_prior_var=None):
+    def __init__(self, X_mean, X_var, Y, kern, M, feat=None, Z=None, X_prior_mean=None, X_prior_var=None):
```

Comment:
Part of the point of the `inducingpoint_wrapper` is to still support people passing arguments by position rather than by name, so `feat=None` should take the position of `Z` and `Z` should move to the end. Alternatively, and maybe preferably, just add the `, *, ` after the `M` to explicitly throw an error?